db/external_sst_file_ingestion_job.cc

```diff
@@ -793,7 +793,20 @@ Status ExternalSstFileIngestionJob::GetIngestedFileInfo(
     file_to_ingest->smallest_internal_key.SetFrom(key);
 
     iter->SeekToLast();
-    pik_status = ParseInternalKey(iter->key(), &key, allow_data_in_errors);
+    auto largest = iter->key();
```

Comment:
iter->key() should be called only if iter is Valid(), which may not be the case for PlainTable here.